### ISSUE TITLE
Map LT/RT to a controller button instead of an axis.

### ISSUE THREAD

**Issue by Renari** [Jays2Kings/DS4Windows#30]
I recently started playing an old game that doesn't recognize the Z axis as a valid input, thus the LT/RT buttons don't work. I'm not sure if it would be possible to map controls to virtual controller buttons.

**Comment by idbrii** [comment]
I think you can create a macro so when you hit LT it fires a button press. (Or maybe only a keystroke.) Did you try macros?

**Comment by Renari** [comment]
Yeah, this works but it would still be nice to be able to do this since a lot of games that ask you to bind controller buttons don't read keyboard inputs in the same program.
